src/SwedbankPay/Api/Client/Version.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+namespace SwedbankPay\Api\Client;
+
+/**
+ * Class Version
+ * @package SwedbankPay\Api\Client
+ */
+class Version
+{
+    private $version;
+
+    /**
+     * Version constructor
+     *
+     * @throws Exception
+     */
+    public function __construct()
+    {
+        $this->version = $this->getVersionFromEnvironment();
+    }
+
+    /**
+     * Gets the version number from a defined constant VERSION, the environment
+     * variable VERSION or from composer.json.
+     *
+     * @return string Version number
+     */
+    public function getVersion()
+    {
+        return $this->version;
+    }
+
+    /**
+     * Gets the version number from a defined constant VERSION, the environment
+     * variable VERSION or from composer.json.
+     *
+     * @return string Version number
+     * @throws ClientException
+     */
+    protected function getVersionFromEnvironment()
+    {
+        if (defined('VERSION')) {
+            return VERSION;
+        }
+
+        if (getenv("VERSION") !== false) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The use of function getenv() is discouraged](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-php/pullRequest?prid=5251412)